alephclient/api.py

```diff
@@ -353,6 +353,32 @@ def _bulk_chunk(
                     return
                 backoff(ae, attempt)
 
+    def write_entity(self, collection_id: str, entity: Iterable, **kw):
+        """Create a single entity via the API, in the given
+        collection.
+
+        params
+        ------
+        collection_id: id of the collection to use. This will overwrite any
+        existing collection specified in the entity
+        entity: A dict object containing the values of the entity
+        """
+        if isinstance(entity, Dict):
```

Comment:
Is this check necessary?

Reply:
mypy complains bitterly if the check isn't there

Reply:
I think that's because of the type annotation in the function signature. `entity` should be a `Dict`, not an `Iterable`. We can also use a custom `Dict` alias like `EntityDict` like @pudo suggested in a comment.